src/js/pages/jobs/JobsTable.js

```diff
@@ -236,7 +236,6 @@ class JobsTable extends React.Component {
   render() {
     return (
       <Table
-        className="table table-borderless-outer table-borderless-inner-columns flush-bottom"
```

Comment:
I think this should still be here. Jobs page looks weird:
![](https://cl.ly/30320E2K2k2S/Image%202016-12-13%20at%2012.04.27.png)